config/database.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  development: {
```

Comment:
I think `docker` is a good env name in current setup. It tells this setting is assuming docker setup and the setting are passing via envvar.

Reply:
sequelize-cli use `development` as config key, need to change `NODE_ENV` or add env argument if want to change this.